idaes/models/properties/general_helmholtz/helmholtz_state.py

```diff
@@ -1037,7 +1108,7 @@ def define_display_vars(self):
                 "P": self.pressure,
                 "Vapor Fraction": self.vapor_frac,
                 "Molar Enthalpy": self.enth_mol,
-                "Molar Entropy": self.entr_mol,
+                # "Molar Entropy": self.entr_mol,
```

Comment:
Is there a reason this was left commented out? Also below.

Reply:
I was thinking about bringing it back but forget.  Will remove.